concepts/blocks/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Introduction
```

Comment:
I'm not really clear on why the about.md is an exact copy of the introduction.md. Maybe there is some reasoning for this, but I don't recall. Is it expected?

Reply:
Usually with more complex topics there is a bigger difference between introductory stuff and details. I usually write the about first and then take a subset for the introduction. Here, though, there didn't seem to be much to take away. Perhaps that means the "about" should be more detailed. But my concern is not to get too much into the weeds with elaborations on scope and such just for the concept of block. A lot of that material is already in the dynamic memory concept.

Reply:
Thank you for looking all of this over!

Reply:
For "about" I added a bit explaining how multiple single-line statements and comments could be one block without needing curly braces. I left that detail out of the introduction.